modules/update-oc-configmap-signature-verification.adoc

```diff
@@ -18,7 +18,7 @@ If you are upgrading from a release prior to version 4.4.8, you must use the man
 
 .Procedure
 
-. Obtain the image signature for the version that you are upgrading to from either link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] or link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)].
+. Obtain the image signature for the version that you are upgrading to from either link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com].
```

Comment:
This flow is broken, even with a working GCS link, because you cannot directly `oc apply ...` the PGP signature.  You need to wrap it in a ConfigMap first.  Folks using recent `oc adm release mirror ...` will have `oc` reach out to the appropriate sig store and produce the sig ConfigMap automarically.  Can we talk about that where we tslk about mirroring (maybe we already do), and here just say:

> `oc` version 4.4.8 and later already mirrored the signature as part of the 
`oc adm release mirror ...` process.  You don't have to do any extra work here.

or some such?  Maybe:

> `oc` version 4.4.8 and later mirror the signature as part of the 
`oc adm release mirror ...` process.  Users with older `oc` can mirror the signature using the procedure described in this section.